file.go

```diff
@@ -0,0 +1,31 @@
+package fyne
+
+import "io"
+
+// FileReader represents a cross platform data stream from a file or provider of data.
+// It may refer to an item on a filesystem or data in another application that we have access to.
+type FileReader interface {
+	io.ReadCloser
```

Comment:
`FileReader` doesn’t read as if it could `Close()`, too.
I think this should be called `FileReadCloser()`.